File: src/CS/Enhanced CS Tabbed Web Browser/FavouriteDirectoryInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Enhanced_CS_Tabbed_Web_Browser
{
    public class FavouriteDirectoryInfo
    {
        public string DirectoryName { get; set; }

        public string FullPath { get; set; }

        public override string ToString()
        {
            return this.DirectoryName;
        }
    }
}

File: src/VB/Enhanced VB Tabbed Web Browser/FavouriteDirectoryInfo.vb
Public Class FavouriteDirectoryInfo
    Private _directoryName As String
    Public Property DirectoryName() As String
        Get
            Return _directoryName
        End Get
        Set(ByVal value As String)
            _directoryName = value
        End Set
    End Property


    Private _fullPath As String
    Public Property FullPath() As String
        Get
            Return _fullPath
        End Get
        Set(ByVal value As String)
            _fullPath = value
        End Set
    End Property

    Public Overrides Function ToString() As String
        Return Me.DirectoryName
    End Function
End Class
